### ISSUE TITLE
error: 'SecCertificateCopyPublicKey' is only available on iOS 10.3 or newer

### ISSUE THREAD

**Issue by kiroskirin** [IBM-Swift/Swift-JWT#64]
I folk this repo and try to make it support Carthage but when I run the build. There is an error like this

```swift
Swift-JWT/.build/checkouts/BlueRSA/Sources/CryptorRSA/CryptorRSAKey.swift:338:15: error: 'SecCertificateCopyPublicKey' is only available on iOS 10.3 or newer

let key = SecCertificateCopyPublicKey(certData)
```

**Comment by Andrew-Lees11** [comment]
Hello @kiroskirin 

There seems to be a mistake in BlueRSA.
At [CryptorRSAKey.swift#L32](https://github.com/IBM-Swift/BlueRSA/blob/master/Sources/CryptorRSA/CryptorRSAKey.swift#L32) it has:
```
@available(macOS 10.12, iOS 10.0, *)
```
Which should be:
```
@available(macOS 10.12, iOS 10.3, *)
```
This might also be the case for other @available methods. 
I will look into this and raise a PR against BlueRSA

**Comment by Andrew-Lees11** [comment]
@kiroskirin 
I have raised pr #67 that should update all the @available lines so that the project compiles on iOS, watchOS and tvOS. Would you be able to test the `available` branch to see if this solves your issue?

**Comment by Andrew-Lees11** [comment]
Closing since this should be fixed in the 3.5.1 release
